notebooks/en/quantize_stable_diffusion_with_quanto.ipynb

```diff
@@ -0,0 +1,1067 @@
+{
```

Comment:
<p>let's again add more and more explanations of what we'll do instead of comments inside the code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>